src/SPCPAuthClient.types.ts

```diff
@@ -0,0 +1,38 @@
+import { XPathSelect } from 'xpath'
+
+export type XpathNode = Parameters<XPathSelect>[1]
```

Comment:
Surely it should be...
```suggestion
export type XPathNode = Parameters<XPathSelect>[1]
```